docs/reference/api-connectors-elasticsearch.md

```diff
@@ -78,8 +78,8 @@ const connector = new ElasticsearchAPIConnector(
 
 | Param             | Type   | Description                                                                                                                                                                                                                                                                                                                                            |
 | ----------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| cloud             | object | **Required if `host` or custom `apiClient` not provided.** Object type. The cloud id for the deployment within elastic cloud. Format: `{ id: 'cloud:id' }`. You can find your cloud id in the Elastic Cloud deployment overview page.                                                                                                                                        |
-| host              | string | **Required if `cloud` or custom `apiClient` not provided.** String type. The host url to the Elasticsearch instance                                                                                                                                                                                                                                                          |
+| cloud             | object | **Required if `host` or custom `apiClient` not provided.** Object type. The cloud id for the deployment within elastic cloud. Format: `{ id: 'cloud:id' }`. You can find your cloud id in the Elastic Cloud deployment overview page.                                                                                                                  |
```

Comment:
is it intended change? Confused since there is no change copy

Reply:
Prettier formatted it, maybe removed some spaces